app/index.js

```diff
@@ -99,11 +98,9 @@ module.exports = yeoman.generators.Base.extend({
 		this.prompt(prompts, function (props) {
 			var componentName = props.componentName;
 
-			var capitalizeName = _.startCase(componentName);
-
 			this.camelCaseName = _.camelCase(componentName);
-			this.capCaseName = capitalizeName.replace(' ', '');
-			this.capitalizeName = capitalizeName;
+			this.capCaseName = componentName;
+			this.capitalizeName = _.startCase(componentName);
```

Comment:
Hey @mairatma, Lodash's startCase() function will add spaces between words as well as capitalize. For instance:

componentName: 'SelectInput'

after startCase():

capitalizeName: 'Select Input'

https://lodash.com/docs/4.16.4#startCase

Right now this is only used on the demo page. As I think about it now, it could very well be personal preference to display the capitalized and spaced name as opposed to the component class name. 

Which would you prefer?